src/components/footer/Footer.tsx

```diff
@@ -29,11 +29,17 @@ const StyledSocialList = styled(Row)`
     justify-content: flex-end;
     align-items: flex-end;
     gap: 4px;
-
+    
     @media (max-width: ${theme.breakpoints.mobileMax}) {
         flex-basis: 100%;
         justify-content: flex-start;
     }
+
+    // To avoid overlap with scroll back to top button as window width gets smaller
+    @media (max-width: 1225px) and (min-width:${theme.breakpoints.mobileMax}) {
+        margin-right: 4rem;
+    }
```

Comment:
It kinda bugs me that we don't have a constant for the first value in the media query ^^ Maybe we can add that later. Also same thing with `px` instead of `rem` as above.